### DISCUSSION TITLE
Horizontal, vertical positions saving, & dependencies?

### DISCUSSION THREAD

**Discussion by andwan0** [YaoFANGUK/video-subtitle-extractor#222] [Q&A] [answered]
Do we need to install the dependencies to run the GPU version? I am not wanting to debug or develop. I just want to run and use the GPU version.

(dependencies eg. CUDA, cuDNN, paddlepaddle?)

Also, is it possible to "save" the vertical, horizontal settings?

**Comment by YaoFANGUK** [comment, marked as answer]
already supported
